isup/isup-impl/src/main/java/org/mobicents/protocols/ss7/isup/util/BcdHelper.java

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright 2016 Grzegorz Figiel
+ *
```

Comment:
Hello,
As I do not know all flavours of the OpenSource licencies I used the license header as in the code I modified https://gist.github.com/neuro-sys/953548 as not to contradict the original license (Apache 2.0). If you know that I can safly change the license into Telestax then I can change the header.